pages/incometax.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <article class="flex flex-col gap-6">
+    <div>
+      <page-title path="incometax.title" />
+      <paragraph>
+        <i18n path="incometax.p1.content" tag="p" class="paragraph-sm xl:paragraph">
+          <a
+          class="italic text-link"
+          :href="'http://static.anaf.ro/static/10/Anaf/legislatie/L_227_2015.htm'"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {{ $t("incometax.p1.law_label") }}
+        </a>
+        </i18n>
```

Comment:
I think this style of link breaks the flow of the text in this case.
![image](https://user-images.githubusercontent.com/1261457/104602682-56640780-5684-11eb-862b-16abfbbd1d02.png)
Linking from the law number directly would not do this.
Also having both a highlighted style and italic does the same thing.
One or both of the above also apply for the other links on the page.
@Traveler3568 @DanChendea

Reply:
Done, talked with @Traveler3568 , agreed to remove italic